tools/nl/embeddings/README.md

```diff
@@ -52,15 +41,17 @@ This is a common sheet across the different index sizes.
 
    To generate the `sdg_ft` embeddings:
    ```bash
-   ./run.sh -c sdg https://docs.google.com/spreadsheets/d/1-QPDWqD131LcDTZ4y_nnqllh66W010HDdows1phyneU SDG_GOALS_INDICATORS_ONLY data/curated_input/sdg_sheets_svs.csv
+   ./run.sh -c sdg data/curated_input/sdg_sheets_svs.csv data/alternatives/sdg/*.csv
    ```
 
    You can also create custom embeddings (using the finetuned model in PROD):
    ```bash
-   ./run.sh -c <embeddings_size> <sheets_url> <worksheet_name> data/curated_input/sheets_svs.csv
+   ./run.sh -c <embeddings_size> <curated_input_csv_path> <alternatives_filepattern>
```

Comment:
nit: how about an example (hypothetical)? That will make clear how to provide relative paths or absolute paths to these files, right? Or do these need to be under secific subpaths?

Reply:
done, added an example in the Notes